pkg/controller/signup.go

```diff
@@ -66,19 +66,37 @@ func (s *Signup) GetHandler(ctx *gin.Context) {
 // VerifyCodeHandler validates the phone verification code passed in by the user
 func (s *Signup) VerifyCodeHandler(ctx *gin.Context) {
 	code := ctx.Param("code")
+	if code == "" {
+		log.Error(ctx, nil, "no code provided in request")
+		ctx.AbortWithStatus(http.StatusNotFound)
```

Comment:
Shouldn't it be 400? And we should also return.
```suggestion
		ctx.AbortWithStatus(http.StatusBadRequest)
		return
```

Reply:
Updated.